### PR TITLE
run hey performance test for container-freezer

### PR DESCRIPTION
**Pull request by jwcesign**
Signed-off-by: jwcesign <jwcesign@gmail.com>

**Which issue(s) this PR fixes**:<br>
Fixes #https://github.com/knative-sandbox/container-freezer/issues/151

### CODE REVIEW

**Comment by jwcesign** [comment]
So can we merge this now? @psschwei

File: prow/jobs_config/knative-sandbox/container-freezer.yaml [hunk 1]
```diff
@@ -33,3 +33,12 @@ jobs:
     command: [runner.sh, ./hack/release.sh, --auto-release, --release-gcs, knative-releases/container-freezer, --release-gcr, gcr.io/knative-releases, --github-token, /etc/hub-token/token]
     requirements: [release]
     excluded_requirements: [gcp]
+
+  - name: performance-tests-hey
```
Comment (psschwei) [inline prow/jobs_config/knative-sandbox/container-freezer.yaml:37]:
```suggestion
  - name: performance-tests
```

**Review by psschwei — COMMENTED** [review]
one nit, otherwise looks good to me

File: prow/jobs_config/knative-sandbox/container-freezer.yaml (hunk 1 above)
Comment (jwcesign) [reply prow/jobs_config/knative-sandbox/container-freezer.yaml]: fixed

**Comment by jwcesign** [comment]
cc @cardil @kvmware, can I get merge?

**Pull request merged**